src/main/java/mate/academy/model/User.java

```diff
@@ -28,4 +38,48 @@ public class User {
     @Column(nullable = false)
     private String lastName;
     private String shippingAddress;
+
+    @ManyToMany(fetch = FetchType.EAGER)
```

Comment:
```suggestion
    @ManyToMany
```
Handle it on the repository level